### ISSUE TITLE
Problem with "Registry: quay.io" in Singularity-file

### ISSUE THREAD

**Issue by TormodLandet** [singularityhub/singularityhub.github.io#10]
I'm trying to bootstrap from a docker image `quay.io/fenicsproject/dev-env:latest` like specified here:
http://singularity.lbl.gov/bootstrap-image#docker

I get an error
~~~
 ERROR:python:Error obtaining tags: https://index.docker.io/v2/fenicsproject/dev-env/tags/list
~~~
When using
~~~
Bootstrap: docker
From: fenicsproject/dev-env:latest
Registry: quay.io
~~~

The singularityfile is here:
https://github.com/TormodLandet/singularity-fenics-dev-env

The log is here:
https://singularity-hub.org/containers/605/log

**Comment by vsoch** [comment]
The version of singularity on shub is unfortunately old, and they carry this bug that has been fixed in newer versions. The good news is that this will be fixed with 2.3 after The builders are updated! Apologies for the hassle, I will let you know when they are updated to repeat the build.

**Comment by TormodLandet** [comment]
OK, thanks for letting me know! You sure are fast to respond :-)

**Comment by vsoch** [comment]
:space_invader:

**Comment by vsoch** [comment]
hey @TormodLandet can you try this out again? The builders are updated to 2.3 - although check out this issue as I've found a bit of inconsistency with downloading quay layers [see here!](https://github.com/singularityware/singularity/issues/749)

**Comment by TormodLandet** [comment]
Closing this since Quay is now supported in v. 2.3. I have not tested on my specific example as I had to leave the idea of using the [FEniCS docker containers](http://fenics-containers.readthedocs.io/en/latest/) directly since they store quite some data in the home directory.

BTW: I made [this tutorial](http://folk.uio.no/tormodla/singularity/) that will be used internally at the math department of the University of Oslo. Feel free to steal from it for your official docs, I could not find any docs on SINGULARITYENV_ there as an example
